textproc/pandoc/Portfile

```diff
@@ -83,10 +83,14 @@ if {(${os.platform} eq {darwin}
         build.target    all
         build.post_args-prepend \
                     {*}${cabalopts}
+        build.post_args-append \
+                 --allow-newer
```

Comment:
This works, but is it the best choice? I have opened upstream fixes for pandoc dependencies (see https://trac.macports.org/ticket/71908#comment:3) that would make this unnecessary, at least until the next version (unless both maintainers agree to `< 5` instead of `< 4.22`; I went conservative for now).

Reply:
It’s the best choice in the sense that it’s the most straightforward way of sidestepping the problem of spending unavailable cycles to determine package- and time-dependent specific versions that fix `cabal` dependency hell. Today’s dependency fix will break tomorrow’s build, so may as well make it a simple fix.